packages/input-option/src/InputOption/InputOption.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  Polymorphic,
+  PolymorphicAs,
+  usePolymorphic,
+} from '@leafygreen-ui/polymorphic';
+
+import {
+  inputOptionActiveStyles,
+  inputOptionDisabledStyles,
+  inputOptionHoverStyles,
+  inputOptionStyles,
+  inputOptionThemeStyles,
+  inputOptionWedge,
+} from './InputOption.style';
+import { InputOptionProps } from './InputOption.types';
+
+export const InputOption = Polymorphic<InputOptionProps>(
+  (
+    {
+      as = 'li' as PolymorphicAs,
+      children,
+      disabled,
+      focused,
```

Comment:
Are we requiring that we set focused in order to know if an option is focused? There's no logic around this internal to the component?

Reply:
No this is a visual component only. The implementing component (Select, Combo, Search) will have it's own logic to determine which option is focused

Reply:
Worth noting that this is different than the native `:focus` state. It's one of the parameters that determines the blue active state

<img width="301" alt="Screen Shot 2023-02-08 at 6 42 00 PM" src="https://user-images.githubusercontent.com/2414030/217676838-363dd02f-7fee-4c46-bf45-2ae1147f3529.png">

Reply:
There are cases (like in Combo) where the `active` (or selected) elements, are different than the `focused` elements. Open to changing this terminology though

Reply:
Maybe `selected` & `highlighted`